O3exp/pipeline/plotting.ini

```diff
@@ -418,11 +418,14 @@ sngl-stat = ${fit_by_template|sngl-stat}
 fit-function = ${fit_by_template|fit-function}
 ; don't try and fit everything down to zero stat value
 stat-fit-threshold = ${fit_by_template|stat-threshold}
+prune-number = 1 
+; note this pruning not the same style as in fit_by_template
+; default 0.1 prune window
```

Comment:
clearer to actually have the option there, even though it is the default - we don't usually put comments on default settings in ini files.